apps/modernization-ui/src/styles/_uswds.scss

```diff
@@ -91,13 +96,20 @@ table.usa-table {
         vertical-align: bottom;
     }
 }
-.usa-button:disabled,
-.usa-button--outline-disabled,
-.usa-button--outline:disabled {
+
+.usa-button:disabled {
     color: colors.$base-white !important;
     background-color: colors.$disabled !important;
     cursor: default;
 }
+
+.usa-button--outline-disabled,
+.usa-button--outline:disabled {
+    background-color: colors.$base-white !important;
+    color: colors.$disabled !important;
```

Comment:
question:  Is the background color of outline buttons supposed to be transparent?

Reply:
going off of the figma it had white, but i guess that was just the background it was placec on.